tests/terraform/checks/provider/panos/test_credentials.py

```diff
@@ -41,5 +44,34 @@ def test_failure_password(self):
         scan_result = check.scan_provider_conf(conf=provider_conf)
         self.assertEqual(CheckResult.FAILED, scan_result)
 
+    def test_api_key(self):
+        runner = Runner()
+        current_dir = os.path.dirname(os.path.realpath(__file__))
+
+        test_files_dir = current_dir + "/resources/api_key"
+        report = runner.run(root_folder=test_files_dir, runner_filter=RunnerFilter(checks=["CKV_PAN_1"]))
```

Comment:
why not import the check and reference the ID like in the other tests?

Reply:
I wanted to debug the variable rendering so I wanted to create a graph

Reply:
```suggestion
        report = runner.run(root_folder=test_files_dir, runner_filter=RunnerFilter(checks=check.id))
```
that's what I meant, didn't see it was already imported 😄

Reply:
yeah you're right, fixed it!